unstructured/ingest/connector/fsspec/gcs.py

```diff
@@ -21,6 +23,30 @@ class GcsAccessConfig(AccessConfig):
         default=None, sensitive=True, overload_name="service_account_key"
     )
 
+    def __post_init__(self):
+        ALLOWED_AUTH_VALUES = "google_default", "cache", "anon", "browser", "cloud"
+        # Case: null value
+        if not self.token:
+            return
+        # Case: one of auth constants
+        if self.token in ALLOWED_AUTH_VALUES:
+            return
+        # Case: token as json
+        try:
+            str_token = self.token.replace("'", '"')
```

Comment:
Why is this replace required? Just curious.

Reply:
I have found that dict is converted to string with single quotes for string type of keys and values. This leads to inability to convert back to JSON.